Esri/arcgis-maps-sdk-swift-samples
# [New] Search for web map

## Description

This PR implements `Search for web map` in `Cloud and Portal` category.
URL to README: [README](https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/Caleb/New-SearchForWebMap/Shared/Samples/Search%20for%20web%20map)

## Linked Issue(s)

- `swift/issues/2260`

## How To Test

- Enter a query into the search bar.
- Tap on a result to see the map for the given portal item.
- Scroll to the bottom of the list to load more results.

## Screenshots

![search-for-web-map](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/34120440/f27b534b-167a-44c7-b2ec-a0102c2dfd5c)
